querybook/webapp/const/keyMap.ts

```diff
@@ -80,6 +80,10 @@ const DEFAULT_KEY_MAP = {
             key: 'Ctrl-D',
             name: 'Delete current cell',
         },
+        onSubmit: {
```

Comment:
can you move this to overallUI and rename it to submitComment?

Reply:
since this also gets shown on the shortcut page, so having submitComment is more clear to the users?